sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverClient.cs

```diff
@@ -349,7 +315,7 @@ protected ServiceBusReceiverClient()
            int maxMessages,
            CancellationToken cancellationToken = default)
         {
-            Argument.AssertNotClosed(IsClosed, nameof(ServiceBusReceiverClient));
+            Argument.AssertNotClosed((bool)this.IsClosed, nameof(ServiceBus.ServiceBusReceiverClient));
```

Comment:
I'm likely overlooking something, but if we have an `IsClosed` member which is returning a bool, I'm not sure why we would need to cast or disambiguate with `this.`

Reply:
I think this was a merge issue